client/extensions/zoninator/state/zones/test/reducer.js

```diff
@@ -157,6 +160,52 @@ describe( 'reducer', () => {
 			} );
 		} );
 
+		it( 'should initiate zones array for a site ID after the first zone is added', () => {
```

Comment:
Hmm, it should indeed create a new array if one doesn't exist yet if that's what you're asking.